cg/models/nf_analysis.py

```diff
@@ -24,11 +25,19 @@ class NextflowSample(BaseModel):
     fastq_forward: List[str]
     fastq_reverse: List[str]
 
+    @validator("fastq_reverse", "fastq_forward")
+    def fastq_not_empty(cls, fastq) -> List[str]:
+        """Verify that fastq files are provided."""
+        if len(fastq) == 0:
```

Comment:
I believe you'll obtain the same results:
```suggestion
        if not fastq:
```

FYI: There is also a pydantic specifuc type, [conlist](https://docs.pydantic.dev/1.10/usage/types/#constrained-types):
`options: conlist(str, min_items=1)`

Reply:
didn't know! ❤️

Reply:
added a constrained list and removed the validation function to assess empty list since it's no longer necessary